content/en/docs/reference/config/analysis/ist0109/index.md

```diff
@@ -0,0 +1,11 @@
+---
+title: ConflictingMeshGatewayVirtualServiceHosts
+layout: analysis-message
+---
+
+This message occurs when Istio detects an overlap between virtual service
+resources. For example, you have multiple virtual services defined to use the
+same hostname and gateway.
```

Comment:
This only applies to sidecars (gateway=mesh or no gateway specified)

Reply:
fixed